+exp/ExpTest.m

```diff
@@ -0,0 +1,330 @@
+% to load a specific parameter set
+% 5) (Optional) Click the 'Options' button to select which options you
+% to enable when running the Exp Def:
+%   a) Live-Plotting
+%   b) Save a 'block' file
+% 6) Click the 'Start' button to run the Exp Def. This button will turn
+% into a 'Stop' button after the experiment starts running - click it again
+% to end the experiment (alternatively, press the 'q' keyboard key to
+% stop the experiment). To run another exp def in the same panel (or the
+% same exp def that was just run), repeat 'Usage' from step 3).
+%
+% *Note: While the experiment is running, the position of the mouse
+% cursor will be set to emulate the wheel. Press the 'c' keyboard key
+% while experiment is running to disable/enable this feature.
+%
+% Todo: set properties and methods attributes appropriately
+  
+  %% properties
+  properties %(Access = private)
+    PanelH % 'ExpTestPanel' figure handle
+    MainGrid % main 'uix.GridFlex' object; child of 'PanelH'
+    ExpGrid % top 'uix.Grid' object; child of 'MainGrid'
+    ExpTopBox % 'uix.HBox' object, containing UI elements to run the Exp Def; child of 'ExpGrid'
+    ExpBottomBox % 'uix.HBox' object, containing UI elements which display experiment info; child of 'ExpGrid'
+    SubjectList % 'bui.Selector' object containing the subject-list
+    Subject % string of currently selected subject (optional)
+    SelectExpDef % handle to 'Select Signals Exp Def' push-button
+    ExpDef % function handle to currently chosen Signals Exp Def
+    ExpDefPath % fullfile path of ExpDef
+    OptionsButton % handle to 'Options' push-button
+    LivePlot = 1 % option for live-plotting signals during experiment
+    SaveBlock = 0 % option for saving 'block' file at experiment end
+    StartButton % handle to 'Start' push-button
+    ParamPanel % 'uix.Panel' object containing the 'ParamGrid'; child of 'MainGrid'
+    ParamGrid % 'uix.GridFlex' object containing 'ParamTopBox' and 'ParamBottomBox'; child of 'ParamPanel'
+    ParamTopBox % 'uix.HBox' object containing 'bui.Selector' object for list of saved parameter sets; child of 'ParamGrid'
+    ParamBottomBox % 'uix.VBox' object containing 'Parameters'; child of 'ParamGrid'
+    Parameters = exp.Parameters % 'exp.Parameters' object containing the current parameters
+    ParametersList % 'bui.Selector' object containing saved list of parameters sets
+    ParametersProfile = '<defaults>' % string of selected parameters set from 'ParametersList'
+    ParamEditor % 'eui.ParamEditor' object for viewing/changing the current parameters
+    RewardCount % uicontrol text object for reward delivered
+    TrialNumCount % uicontrol text object for trial number
+    IsRunning = 0 % flag for if Exp Def is running
+    ScreenH % handle to PTB Screen which displays visual stimuli
+    LivePlotFig % handle to figure for live-plotting signals
+    
+    SignalsExpTestObj % 'SignalsExpTest' object which contains info for running the *Signals* experiment
```

Comment:
Avoid calling the property an object.  SignalsExpTest would suffice.  Also perhaps a different name might be better to avoid a conflict with our future test suite?  I don't know what would be best...

Reply:
why avoid calling it an object? When a property is to be an object of a certain class, I find it helpful to read that in the property's declaration/description.

Reply:
ah i see your point below, fair enough